src/components/molecules/section/Section.tsx

```diff
@@ -1,35 +1,15 @@
 import { ReactNode } from "react";
 
-import { joinTailwindClasses } from "@/utils/tailwind";
-
-import { Badge } from "../../atoms/label/Badge";
-
 export interface IProps {
   children: ReactNode;
-  className?: string;
-  controls?: ReactNode;
-  explanation?: ReactNode;
-  title: string | ReactNode;
-  titleBadge?: string;
+  id: string;
+  title?: string;
```

Comment:
Question: does limiting the title to be a string limit us to adding things like tags/badges whatever later?